src/techCompanies/orderByAlphabeth/c.js

```diff
@@ -84,13 +84,20 @@ export default {
       foundersTwitterHandle: [ ] ,
       industry: "Fintech",
       website: "https://credbevy.com",
-    }
+    },
     {
       companyName: "CokoSofts",
       companyTwitterHandle: "cokosofts",
       foundersTwitterHandle: ["oby"],
       industry: "Software Development",
       website: "https://cokosofts.com/",
     },
+    {
+      companyName: "Cue",
+      companyTwitterHandle: "https://linkedin.com/company/cue-technologies-ltd/",
```

Comment:
This is not an ideal twitter link. I believe it's best to just leave it as an empty string. When the company has an official twitter account in the future, it can be added in.